res/values/strings.xml

```diff
@@ -465,6 +465,28 @@
     <!-- the explanation is shown (1) as a modal dialog with the buttons "Cancel" and "Continue" as well as (2) as a device message -->
     <string name="aeap_explanation">You changed your email address from %1$s to %2$s.\n\nIf you now send a message to a verified group, contacts there will automatically replace the old with your new address.\n\nIt\'s highly advised to set up your old email provider to forward all emails to your new email address. Otherwise you might miss messages of contacts who did not get your new address yet.</string>
 
+
+    <!-- Multi Device -->
+    <string name="multidevice_title">Add Another Device</string>
+    <string name="multidevice_same_network_hint">Make sure both devices are on the same Wi-Fi or network</string>
+    <string name="multidevice_install_dc_on_other_device">Install and start Delta Chat 1.36 or newer on your other device (https://get.delta.chat)</string>
+    <string name="multidevice_tap_scan_on_other_device">During setup, tap “Scan QR Code” and scan the code shown here</string>
+    <!-- Shown inside a "QR code card" with very limited space; please formulate the text as short as possible therefore. The placeholder will be replaced by name and/or address eg. "Scan to set up another device for Alice (alice@example.org)" -->
+    <string name="multidevice_qr_subtitle">Scan to set up another device for %1$s</string>
+    <string name="multidevice_receiver_scanning_ask">Copy the account from the other device to this device? Both devices will work independently afterwards.</string>
+    <string name="multidevice_exporting_database">Exporting database…</string>
+    <string name="multidevice_creating_collection">Creating collection…</string>
```

Comment:
not sure about `Creating collection / Collection created / Receiving Collection / Collection received` which is "conceptually the same thing as creating a tar file" (@dignifiedquire)

on alternative idea was `Preparing content` - but there `Receiving content` does not really work (as afterwards it takes minutes to get the real content)

`Preparing your account data for transfer to second device`,  `Preparing account data for export` was another idea, but a but long (i think, it should be really short as of now).

my alternative idea was `Creating index / Index created / Receiving index / Index received`, btw, as "Index" seems to be a more known thing and easier to translate - still not sure if that would fit :)

Reply:
`Creating collection…` should be absolutely changed for something else, maybe:
* Preparing for transfer…
* Getting ready for transfer…
* Preparing account for transfer…

Reply:
what would be "receiving colllection"/"collection received" then? "preparing to receive..."/"receiving prepared"?

Reply:
maybe just say "Preparing account..."/"Account prepared." on both sides? and also change to "Export account." the term "Account" is pretty well used already and we would not add a new term.

Reply:
go for that for now, see https://github.com/deltachat/deltachat-android/pull/2493/commits/46871bcbc1365b83d683ddd5b823ddcc1ecd5130